iidm/iidm-xml-converter/src/test/java/com/powsybl/iidm/xml/GeographicalTagsTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.xml;
+
+import com.powsybl.commons.config.InMemoryPlatformConfig;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.datasource.MemDataSource;
+import com.powsybl.iidm.network.Network;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * @author Jérémy Labous <jlabous at silicom.fr>
+ */
+public class GeographicalTagsTest extends AbstractXmlConverterTest {
+
+    private void geographicalTagsTest(Network network, IidmXmlVersion version) throws IOException {
+        PlatformConfig platformConfig = new InMemoryPlatformConfig(fileSystem);
```

Comment:
You do not need to create a `PlatformConfig` object. Simply call `new XmlExporter()`